cvmfs/catalog_mgr_impl.h

```diff
@@ -581,6 +581,23 @@ bool AbstractCatalogManager<CatalogT>::ListFileChunks(
   return result;
 }
 
+template <class CatalogT>
+catalog::Counters AbstractCatalogManager<CatalogT>::LookupCounters(
+  const PathString &path,
+  std::string *subcatalog_path)
+{
+  catalog::DirectoryEntry dummy_dirent;
+  PathString dummy_path = path;
+  dummy_path.Append("/f", 2);  // any filename is good
+  LookupPath(dummy_path, catalog::kLookupSole, &dummy_dirent);
```

Comment:
There is still a race, where between `LookupPath` and `ReadLock` everything can change (e.g. complete repository reload).  I think this method needs to follow the pattern of the other public methods

```C++
EnforceSqliteMemLimit();
ReadLock();

CatalogT *best_fit = FindCatalog(path);
CatalogT *catalog = best_fit;
if (MountSubtree(path, best_fit, NULL)) {
  Unlock();
  WriteLock();
  // Check again to avoid race
  best_fit = FindCatalog(path);
  result = MountSubtree(path, best_fit, &catalog);
  if (!result) {
    Unlock();
    return false;
  }
}

// DO STUFF

Unlock();
```